TROUBLESHOOTING.md

```diff
@@ -193,7 +193,20 @@ only be enabled for debugging.
   moving to the next step.
 
 * ```XLA_USE_BF16```: If set to 1, tranforms all the _PyTorch_ _Float_ values into _BiFloat16_
-  when sending to the _TPU_ device.
+  when sending to the _TPU_ device. Note that when using `XLA_USE_BF16=1` tensor arithmetic will
+  be done in reduced precision and so tensors such as loss values and other metrics will not be
+  useful if accumulated over time. For example:
+  
+  ```
+  # In reduced bfloat16 precision
+  >>> torch.tensor(4096, dtype=torch.bfloat16) + torch.tensor(1, dtype=torch.bfloat16)
+  tensor(4096., dtype=torch.bfloat16)
+  # Whereas in full float32 precision
+  >>> torch.tensor(4096) + torch.tensor(1)
+  tensor(4097)
+  ```
+  So to get accurate loss and other metric values, user manual mixed precision where weights
```

Comment:
typo: user -> use

Reply:
weights staying in bf16 should not be that big of a problem.

Reply:
"so to get metric values such as average loss value over many steps, ..."

Reply:
It does cause some convergence issues, but agreed it's not as big of deal so I'll delete that comment.